### PR TITLE
Feat/ui/restore-should-fetch-images

### PR DESCRIPTION
**Pull request by psychedelicious**
- Try once to get new image URLs if they error on load
- If fails again, direct user to reload page

### CODE REVIEW

**Comment by psychedelicious** [comment]
@maryhipp

File: invokeai/frontend/web/src/features/canvas/util/konvaNodeToBlob.ts [hunk 1]
```diff
@@ -12,5 +12,6 @@ export const konvaNodeToBlob = async (
   node: Konva.Node,
   boundingBox: IRect
 ): Promise<Blob> => {
+  console.log(node);
```
Comment (maryhipp) [inline invokeai/frontend/web/src/features/canvas/util/konvaNodeToBlob.ts:15]: Should remove this?

**Comment by maryhipp** [comment]
I pushed up a fix for txt2img and img2img but am still having trouble with canvas. I think the key is that `setInitialCanvasImage` is using whatever image was in the list when the component loads and isn't updating with the new URL when they are refetched. So when image is sent to canvas, it uses the stale version of image

**Comment by maryhipp** [comment]
I pushed up a fix for txt2img and img2img but am still having trouble with canvas. I think the key is that `setInitialCanvasImage` is using whatever image was in the list when the component loads and isn't updating with the new URL when they are refetched. So when image is sent to canvas, it uses the stale version of image

**Comment by maryhipp** [comment]
This still isn't working for me on canvas 😢

**Pull request closed without merging**